src/model/webviewPages.ts

```diff
@@ -29,6 +29,8 @@ export interface IDependencyPage {
 	license?: ILicense[]
 	watchName?: string[]
 	impactGraph: IImpactGraph
+	pathsCount?: number
+	pathsLimit?: number
```

Comment:
Without the `impactGraph`, both variables lack meaningful context. New readers might find it hard to understand their purpose, and they would need to search for them in the codebase. As both variables are closely connected to the `impactGraph`, it will make sense to create a new interface that binds them together with an impact path.